extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -315,8 +342,19 @@ public synchronized boolean isRunning() {
 
         public synchronized void setRunning(boolean running) {
```

Comment:
Maybe this need to be package private ?

Reply:
+1. In fact, the same applies to `isRunning()`.

Reply:
done